rails/app/decorators/place_photo_decorator.rb

```diff
@@ -4,7 +4,7 @@ def initialize(filename)
   end
 
   def attachable?
-    File.exist?(path)
+    @filename != "" && File.exist?(path)
```

Comment:
in ruby, there's a preference for a method that catches all the empty-variations
@filename.present? 
https://blog.appsignal.com/2018/09/11/differences-between-nil-empty-blank-and-present.html

Reply:
```
@filename.present? && File.exist?(path)
```